wct.conf.js

```diff
@@ -1,14 +0,0 @@
-module.exports = {
```

Comment:
@giotramu the `wct.config.js` should not be removed (at least until all the wct tests are not removed)

Reply:
Restored.